crates/apps_lib/src/cli.rs

```diff
@@ -5164,6 +5148,8 @@ pub mod args {
             ctx: &mut Context,
         ) -> Result<TxIbcTransfer<SdkTypes>, Self::Error> {
             let tx = self.tx.to_sdk(ctx)?;
+            let shielded_sync =
+                self.shielded_sync.map(|sync| sync.to_sdk(ctx).unwrap());
```

Comment:
I think these shielded sync args are missing in `TxOsmosisSwap` and `GenIbcShieldingTransfer`